tests/app/questionnaire/test_rules.py

```diff
@@ -97,6 +97,37 @@ def test_evaluate_rule_equals_with_number(self):
         self.assertFalse(evaluate_rule(when, 2))
         self.assertTrue(evaluate_rule(when, 0))
 
+    def test_evaluate_rule_equals_with_string(self):
```

Comment:
Think I would like to see something referencing the case difference in the test name or a comment